documentation/library_guide/api_for_sycl_kernels/tested_standard_cpp_api.rst

```diff
@@ -459,7 +459,7 @@ libc++ (LLVM)                                 Provided with Clang*-11.0
 --------------------------------------------- ---------------------------------------------
 Microsoft Visual C++* (MSVC) Standard Library Provided with Microsoft Visual Studio* 2017;
```

Comment:
@dmitriy-sobolev Which tested versions of C++ we should specify here?

Reply:
Let's review the versions to note here separately. We definitely should update them, e.g. VS 2017, which I see here, is not supported anymore.